hls-test-utils/src/Test/Hls.hs

```diff
@@ -322,9 +325,27 @@ mkPluginTestDescriptor'
   -> PluginTestDescriptor b
 mkPluginTestDescriptor' pluginDesc plId _recorder = IdePlugins [pluginDesc plId]
 
--- | Initialise a recorder that can be instructed to write to stderr by
--- setting the environment variable "HLS_TEST_PLUGIN_LOG_STDERR=1" before
--- running the tests.
+-- | Initialize a recorder that can be instructed to write to stderr by
+-- setting one of the environment variables:
+-- "HLS_TEST_HARNESS_STDERR", "HLS_TEST_LOG_STDERR"
```

Comment:
```suggestion
-- "HLS_TEST_HARNESS_STDERR=1", "HLS_TEST_LOG_STDERR=1"
```

Since `HLS_TEST_HARNESS_STDERR=0` will actually not work.

Perhaps it would be even nicer if the comment looked like:
```suggestion
-- * HLS_TEST_HARNESS_STDERR=1
-- * HLS_TEST_LOG_STDERR=1
```